src/new_hope/week1/day2.clj

```diff
@@ -0,0 +1,59 @@
+(ns new-hope.week1.day2
+  (:require [midje.sweet :refer :all]
+            [criterium.core :as crit]))
+
+
+;The	rest	function	will	return	all	the	items	of	a	sequence	except	the	first.
+(facts "rest func"
+       (rest  [10  20  30  40]) => [20 30 40])
+
+
+;Intro	to	Functions
+;Clojure	has	many	different	ways	to	create	functions.
+
+(facts "func 1"
+       8 =>  ((fn  add-five  [x]  (+  x  5))  3))
+(facts "func 2"
+       8 => ((fn  [x]  (+  x  5))  3))
+(facts "func 3"
+       8 => (#(+ %  5)  3))
+(facts "func 4"
+       8 =>((partial  +  5)  3))
+
+;Write	a	function	which	doubles	a	number.
+(defn num-double [x] (* x 2))
```

Comment:
facts should use `=>`, otherwise no assertions are made, the result is always "pass"